src/AppBundle/Command/AddUserCommand.php

```diff
@@ -298,4 +292,66 @@ private function getCommandHelp()
 
 HELP;
     }
+
+    /**
+     * Gets the value of entityManager.
+     *
+     * @return ObjectManager
+     */
+    public function getEntityManager()
+    {
+        return $this->entityManager;
+    }
+
+    /**
+     * Sets the value of entityManager.
+     *
+     * @param ObjectManager $entityManager the entity manager
+     *
+     * @return self
+     */
+    public function setEntityManager(ObjectManager $entityManager)
+    {
+        $this->entityManager = $entityManager;
+
+        return $this;
+    }
+
+    /**
+     * Sets the value of entityManager.
+     *
+     * @param ObjectManager $entityManager $entityManager the entity manager
+     *
+     * @return self
+     */
+    private function _setEntityManager($entityManager)
+    {
+        $this->entityManager = $entityManager;
+
+        return $this;
+    }
+
+    /**
+     * Gets the value of passwordEncoder.
+     *
+     * @return UserPasswordEncoder $passwordEncoder
+     */
+    public function getPasswordEncoder()
```

Comment:
Adding a public getter for the dependency of the command does not make any sense. There is no reason for a service to expose its dependencies to the outside